internal/controller/management_controller.go

```diff
@@ -398,6 +431,7 @@ func (r *ManagementReconciler) cleanupRemovedComponents(ctx context.Context, man
 		}
 
 		l.Info("Found component to remove", "component_name", componentName)
```

Comment:
```suggestion
		l.V(1).Info("Found component to remove", "component_name", componentName)
```

could be now debug logs i suppose (and probably some of the others that might emit too many log entries)

Reply:
Unresolved. Sorry for not being specific, my idea was to reduce some log entries in this particular method/loop. There is not much sense in suppressing error messages for the production logger, I think even most of the info messages such as `created <some entity>` are useful in such a manner. Could you please revert most of the changes? Once again, sorry for not specifying the suggestion